File: devpractice/dotnet/oops&collections/EmployeeManagement/Developer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace EmployeeManagement
{
    public class Developer : Employee
    {
        public string _devLang;
        public Developer(string devLang, string name, int empId) : base(name, empId)
        {
            _devLang = devLang;
        }

        public override void workToDo()
        {
            Console.WriteLine($"Developer writes code in {_devLang}");
        }
    }
}

File: devpractice/java/employee_management/src/main/java/org/example/Developer.java
package org.example;

public class Developer extends Employee{
    private String lang;
    public Developer(String lang, String name, int empId){
        super(name, empId);
        this.lang = lang;
    }

    @Override
    public void work() {
        System.out.println(this.name + " develops using language "+ this.lang);
    }
}
